### DISCUSSION TITLE
Get autoincrement as integer

### DISCUSSION THREAD

**Discussion by DenisSteff** [benkeen/generatedata#700] [Ideas]
Hello,

your tool is very useful !!

I generate a dataset for my c# application, but my auto increment id is generated as string. 

I'd like to have an option to generate it as integer

Regards

**Comment by benkeen** [comment]
Absolutely! The C# Export Type just needs updating to handle the Auto-Increment value properly. I'll include this in the next release.

**Reply by benkeen** [reply]
https://github.com/benkeen/generatedata/issues/701
